content/influxdb/v2.0/get-started.md

```diff
@@ -330,14 +330,31 @@ docker run -p 8086:8086 quay.io/influxdb/influxdb:v2.0.4 --reporting-disabled
 ```
 {{% /note %}}
 
-### Console into the InfluxDB Container (Optional)
+### Console into the InfluxDB container (optional)
 
 To use the `influx` command line interface, console into the `influxdb` Docker container:
 
 ```bash
 docker exec -it influxdb /bin/bash
 ```
 
+### Persist data outside the InfluxDB container (optional)
```

Comment:
I don’t know that’d I’d even say this is optional. Mounting a local directory should be the default behavior we suggest. If they don’t specify a local mount, they will lose all their data any time the container restarts.